bin/data-get.js

```diff
@@ -37,8 +37,9 @@ const run = async () => {
   try {
     const start = new Date()
     let savedPath
+    const itIsDatahubOrGitHub = parseDatasetIdentifier(identifier)
     const itIsDataset = isDataset(identifier)
-    if (itIsDataset) {
+    if (itIsDataset || itIsDatahubOrGitHub.type === "datahub" || itIsDatahubOrGitHub.type === "github") {
```

Comment:
so here you'd need to change variable name `itIsDatahubOrGitHub` => `parsedIdentifier`